### PR TITLE
feat: Add Verify presentation endpoint

### LINKED ISSUE #98: Add Verify presentation endpoint

### PR DESCRIPTION
**Pull request by talwinder50**
closes #98

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

**Comment by talwinder50** [comment]
> need bdd test
Ok will add bdd test

File: pkg/restapi/vc/operation/operations.go [hunk 1]
```diff
@@ -642,3 +674,18 @@ func (o *Operation) parseAndVerifyVC(vcBytes []byte) (*verifiable.Credential, er
 
 	return vc, nil
 }
+
+func (o *Operation) parseAndVerifyVP(vpBytes []byte) (*verifiable.Presentation, error) {
+	vp, err := verifiable.NewPresentation(
```
Comment (sandrask) [inline pkg/restapi/vc/operation/operations.go]: you can return here

File: pkg/restapi/vc/operation/operations.go [hunk 2]
```diff
@@ -642,3 +674,18 @@ func (o *Operation) parseAndVerifyVC(vcBytes []byte) (*verifiable.Credential, er
...
+		verifiable.WithPresEmbeddedSignatureSuites(ed25519signature2018.New()),
+		verifiable.WithPresPublicKeyFetcher(
+			verifiable.NewDIDKeyResolver(o.vdri).PublicKeyFetcher(),
+		),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return vp, nil
```
Comment (sandrask) [inline pkg/restapi/vc/operation/operations.go]: Should you check credential status here (call verifyVC)?

File: pkg/restapi/vc/operation/operations.go (hunk 1 above)
Comment (talwinder50) [reply pkg/restapi/vc/operation/operations.go]: due to  https://github.com/trustbloc/edge-service/pull/99#discussion_r388032493 comment, no longer its applied here.

File: pkg/restapi/vc/operation/operations.go (hunk 2 above)
Comment (talwinder50) [reply pkg/restapi/vc/operation/operations.go]: added that

Comment (sandrask) [reply pkg/restapi/vc/operation/operations.go]: looks good

File: test/bdd/pkg/context/context.go [hunk 3]
```diff
@@ -55,9 +56,25 @@ func NewBDDContext() (*BDDContext, error) {
   },
   "profile": "ToBeChangedInStep"
 }`),
+		// nolint: lll
+		CreatedPresentation: []byte(`{
```
Comment (fqutishat) [inline test/bdd/pkg/context/context.go]: use CreatedCredential to create presentation

**Pull request merged**